<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QDialog" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="closeForm">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#closeForm {
   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color:rgb(44, 47, 51);
    font: bold 25px;
    padding: 1px;
}
QPushButton#closeForm:pressed {
    background-color: rgb(88, 101, 242,);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>âœ…</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
 font: bold 20px;
border-width: 2px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INFO FORM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="high_peeker">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: bold 20px;
border-width: 2px;
border-radius: 15px;
background-color: rgb(46, 48, 66);</string>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 101, 242);
border-style: outset;
border-width: 3px;
border-radius: 20px;
border-color: rgb(46, 48, 66);
font: bold 14px;
padding: 1px;

</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>closeForm</zorder>
  <zorder>info_label</zorder>
  <zorder>high_peeker</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
